doc/source/kubecluster2.rst

```diff
@@ -0,0 +1,106 @@
+KubeCluster2
+===========
+
+:doc:`KubeCluster2` is for creating and managing a Dask cluster which using a Kubernetes Operator. With this cluster object you can conveniently create and manage a Dask :class:`dask.distributed.Client` object to the cluster and perform your work. Provided you have API access to Kubernetes and can run the ``kubectl`` command then connectivity to the Dask cluster is handled automatically for you via services or port forwarding.
+
+Installing the Operator
+-----------------------
+
+.. currentmodule:: dask_kubernetes
+
+First you must install the Dask Operator and have access to a Kubernetes Cluster. To install the the operator, first you'll need to deploy the custom resources
+
+.. code-block:: bash
+
+   kubectl apply -f dask_kubernetes/operator/customresources/daskcluster.yaml
+   kubectl apply -f dask_kubernetes/operator/customresources/daskworkergroup.yaml
+
+Finally, you'll need to create the operator
+
+.. code-block:: bash
+   kubectl apply -f dask_kubernetes/operator/deployment/manifest.yaml
+   
+Creating a Dask cluster via `kubectl`
+-------------------------------------
+
+To create a cluster in the default namespace, run the following
+
+.. code-block:: bash
+
+   kubectl apply -f dask_kubernetes/operator/tests/resources/simplecluster.yaml
```

Comment:
Instead of having them install the example from the tests we should show an example of the YAML file here and describe the different configuration options.